spec/serializers/comment_serializer_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe CommentSerializer do
+  let(:debate) { create(:debate) }
+  let(:auth_token) { debate.auth_tokens.create }
+  let(:mobile_user) { create(:edge_mobile_user, auth_token: auth_token, first_name: 'John', last_name: 'Doe') }
```

Comment:
Wow, nie wiedziałem że mamy też "edge" modele. Moim zdaniem to overkill, jeżeli nie potrafimy ogarnąć różnicy pomiędzy API normalnym i edgowym w kontrolerach, to powinniśmy jeszcze raz przemyśleć to rozwiązanie.